keel-test/src/main/kotlin/com/netflix/spinnaker/keel/test/artifacts.kt

```diff
@@ -25,7 +25,7 @@ class DummyArtifact(
 }
 
 object DummySortingStrategy : SortingStrategy {
-  override val comparator: Comparator<PublishedArtifact> = compareBy { it.version }
+  override val comparator: Comparator<PublishedArtifact> = compareByDescending { it.version }
```

Comment:
This was the pain in all of this comparator mess!

Reply:
:( :( :( :( :( im so glad you figured this out